src/amplitude-client.js

```diff
@@ -1258,8 +1258,8 @@ AmplitudeClient.prototype._limitEventsQueued = function _limitEventsQueued(queue
  * Note: the server response code and response body from the event upload are passed to the callback function.
  * @example amplitudeClient.logEvent('Clicked Homepage Button', {'finished_flow': false, 'clicks': 15});
  */
-AmplitudeClient.prototype.logEvent = function logEvent(eventType, eventProperties, opt_callback) {
-  if (this._shouldDeferCall()) {
+AmplitudeClient.prototype.logEvent = function logEvent(eventType, eventProperties, opt_callback, forceDiffer) {
+  if (this._shouldDeferCall() || forceDiffer) {
```

Comment:
I don't think the logic of what you're adding is quite right - it looks like you're pushing the call to `_q` to force a defer, but my understanding is that `_q` is meant to facilitate the proxy object and is only cleared/called when we finish initializing the SDK - this might never be called.